assets/onboarding/data/actions.js

```diff
@@ -126,10 +126,12 @@ export const errorSubmit = ( error ) => ( {
 /**
  * Submit step action creator.
  *
- * @param {string} step     Step name.
- * @param {Object} stepData Data to submit.
+ * @param {string}   step      Step name.
+ * @param {Object}   stepData  Data to submit.
+ * @param {Function} onSuccess Step name.
+ * @param {Function} onError   Data to submit.
  */
-export function* submitStep( step, stepData ) {
+export function* submitStep( step, stepData, onSuccess, onError ) {
```

Comment:
We could group the new parameters into an `options` object, and use the `{  onSuccess, onError }` syntax - a bit cleaner with less function parameters, and easier when someone only wants to add an `onError` handler.

Reply:
Good idea! https://github.com/Automattic/sensei/pull/3184/commits/a5b1afb7f9aecf4b2e9746c1e366b1d45181b777